File: task 3.js
Red_button = findViewById(R.id.Red_button);

        Red_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                TextView textView = findViewById(R.id.editTextTextPersonName);
                String text = "Name";
                SpannableString ss = new SpannableString(text);
                ForegroundColorSpan fcsBlack = new ForegroundColorSpan(Color.Black);
                ss.setSpan(fcsBlack,0,4, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
                textView.setText(ss);
            }
        });
       Size1_button = findViewById(R.id.Size1_button);

        Size1_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                TextView textView = findViewById(R.id.editTextTextPersonName);
                String text = "Name";
                SpannableString ss = new SpannableString(text);
                ForegroundColorSpan fcsBlack = new ForegroundColorSpan(Color.Black);
                ss.setSpan(fcsBlack,0,4, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
                textView.setText(ss);
            }

        });
        Size2_button = findViewById(R.id.Size2_button);

        Size2_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                TextView textView = findViewById(R.id.editTextTextPersonName);
                String text = "Name";
                SpannableString ss = new SpannableString(text);
                ForegroundColorSpan fcsBlack = new ForegroundColorSpan(Color.Black);
                ss.setSpan(fcsBlack,0,4, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
                textView.setText(ss);
            }

        });
        White_button = findViewById(R.id.White_button);

        White_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                TextView textView = findViewById(R.id.editTextTextPersonName);
                String text = "Name";
                SpannableString ss = new SpannableString(text);
                ForegroundColorSpan fcsBlack = new ForegroundColorSpan(Color.Black);
                ss.setSpan(fcsBlack,0,4, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
                textView.setText(ss);
            }

        });
        Yellow_button = findViewById(R.id.Yellow_button);

        Yellow_button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                TextView textView = findViewById(R.id.editTextTextPersonName);
                String text = "Name";
                SpannableString ss = new SpannableString(text);
                ForegroundColorSpan fcsBlack = new ForegroundColorSpan(Color.Black);
                ss.setSpan(fcsBlack,0,4, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
                textView.setText(ss);
            }

        });
    }

File: task 3.kt
package com.example.attributes

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val text: EditText = findViewById(R.id.text)
        val blackButton: Button = findViewById(R.id.blackButton)
        val redButton: Button = findViewById(R.id.redButton)
        val size1Button: Button = findViewById(R.id.size1Button)
        val size2Button: Button = findViewById(R.id.size2Button)
        val whiteButton: Button = findViewById(R.id.whiteButton)
        val yellowButton: Button = findViewById(R.id.yellowButton)

        blackButton.setOnClickListener {
            text.setTextColor(Color.BLACK) }

        redButton.setOnClickListener {
            text.setTextColor(Color.RED) }

        size1Button.setOnClickListener {
            text.textSize = 8f
        }

        size2Button.setOnClickListener {
            text.textSize = 24f
        }

        whiteButton.setOnClickListener {
            text.setBackgroundColor(Color.WHITE)
        }

        yellowButton.setOnClickListener {
            text.setBackgroundColor(Color.YELLOW)
        }
    }
}
